docs/tutorials/tutorial2.md

```diff
@@ -14,148 +14,49 @@ As reported in our paper, we trained an AtacWorks model to learn a mapping from
-```
-mkdir configs
-wget -P configs https://atacworks-paper.s3.us-east-2.amazonaws.com/dsc_atac_blood_cell_denoising_experiments/50_cells/configs/infer_config.yaml
-wget -P configs https://atacworks-paper.s3.us-east-2.amazonaws.com/dsc_atac_blood_cell_denoising_experiments/50_cells/configs/model_structure.yaml
-```
-
-## Step 4: Download the test dsc-ATAC-seq signal from 50 NK cells (~1M reads), in bigWig format
-
-```
-wget https://atacworks-paper.s3.us-east-2.amazonaws.com/dsc_atac_blood_cell_denoising_experiments/50_cells/test_data/noisy_data/dsc.1.NK.50.cutsites.smoothed.200.bw
-```
-
-## Step 5: Create genomic intervals to define regions for testing
-
-The model we downloaded takes the input ATAC-seq signal in non-overlapping genomic intervals spanning 50,000 bp. To define the genomic regions for the model to read, we take the chromosomes on which we want to apply the model and split their lengths into 50,000-bp intervals, which we save in BED format. 
-In this example, we will apply the model to chromosomes 1-22. The reference genome we use is hg19. We use the prepared chromosome sizes file `hg19.auto.sizes`, which contains the sizes of chromosomes 1-22 in hg19.
-```
-mkdir intervals
-python $atacworks/scripts/get_intervals.py \
-    --sizes $atacworks/data/reference/hg19.auto.sizes \
-    --intervalsize 50000 \
-    --out_dir intervals \
-    --prefix hg19.50000 \
-    --wg
-```
-This produces a BED file (`intervals/hg19.50000.genome_intervals.bed`).
-
-For more information type `python $atacworks/scripts/get_intervals.py --help`
-
-## Step 6: Read test data over the selected intervals, and save in .h5 format
-
-We supply to `bw2h5.py` the bigWig file containing the noisy ATAC-seq signal, and the BED file containing the intervals on which to apply the model. This script reads the ATAC-seq signal within each supplied interval and saves it to a .h5 file.
-
-```
-python $atacworks/scripts/bw2h5.py \
-           --noisybw dsc.1.NK.50.cutsites.smoothed.200.bw \
-           --intervals intervals/hg19.50000.genome_intervals.bed \
-           --out_dir ./ \
-           --prefix NK.50_cells \
-           --pad 5000 \
-           --nolabel
-```
-This creates a file `NK.50_cells.h5`, which contains the noisy ATAC-seq signal to be fed to the pre-trained model.
-
-For more information type `python $atacworks/scripts/bw2h5.py --help`
+See [pre-trained denoising models](pretrained_models.md) for a list of the available pre-trained denoising models.
 
-## Step 7: Inference on selected intervals, producing denoised track and binary peak calls
+Before using one of these models, please read the description of how the training datasets for these models were preprocessed, in [Lal & Chiang, et al. (2019)](https://www.biorxiv.org/content/10.1101/829481v2), Methods section, paragraph 1. If your data processing pipeline is different, it is advisable to train a new model using the instructions above.
```

Comment:
Rewriting this line:
Before using one of these models, please consider the following:
1. Please read the description of how the training datasets for these models were preprocessed, in [Lal & Chiang, et al. (2019)](https://www.biorxiv.org/content/10.1101/829481v2), Methods section, paragraph 1. If your data processing pipeline is different, it is advisable to train a new model using the instructions in Tutorial 1 (link).
2. AtacWorks models should only be applied to data of similar sequencing depth as the data used to train the model. If the sequencing depth of your dataset is different from that used for our pre-trained models, it is also advisable to train a new model using the instructions in Tutorial 1 (link).